docs/standard/terminology.md

```diff
@@ -0,0 +1,69 @@
+
+
+## 2. Definition
+
+When new terms are added to the working document, subject matter experts (SME) in the source language are noticed, and write a definition for each term. The definitions must be bound to the context of the source documents, and not general.
+
+Using other terms of the glossary in the definitions is a good practice and helps contextualizing the terms.
+
+If the definition is copied from an existing publication, please add the URL or reference at the end of the definition.
+
+| #   | Step name  | Description                                                                           | Tool               |
+| --- | ---------- | ------------------------------------------------------------------------------------- | ------------------ |
+| 1   | Definition | The source language SME writes a definition for each term, or copies an existing one. | Google spreadsheet |
+
+
+
+## 3. Translation
+
+When the definition of the new terms are ready, subject matter experts (SME) are noticed. They only translate terms to their native language.
+
+The SME may add a comment in the `comment_xx-XX` column to explain the choice of the term. The comment must be written in the target language.
+
+The SME pays attention to the definition given for the term, as it may narrow down the meaning and lead to a translation that is more specific than the usual translation of the source term.
+
+
+| #   | Step name   | Description                                                                           | Tool               |
+| --- | ----------- | ------------------------------------------------------------------------------------- | ------------------ |
+| 1   | Translation | The SME translates the terms to the target language and adds comments when necessary. | Google spreadsheet |
+
+## 4. Proofreading and editing
+
+When the SME is done transating the new terms (or trying to), the language owner enforces a list of rules to maintain the readibility of the terms and the associated comments.
+
+| #   | Step name    | Description                                                                                                               | Tool               |
+| --- | ------------ | ------------------------------------------------------------------------------------------------------------------------- | ------------------ |
+| 1   | Translations | The language owner checks the translations are well spelt and written in lower case (proper nouns and acronyms excepted). | Google spreadsheet |
+| 2   | Comments     | The language owner makes sure the comments are well spelt and written as full sentences in sentence case                  | Google spreadsheet |
+
+
+## 5. Publication
+
+When the translations are proofread and edited, the language owner downloads the Google spreadsheet as a CSV file and replaces the previous file in the Github repository.
```

Comment:
We refer elsewhere to the "Google spreadsheet" as a "working document", and we only use "Google spreadsheet" in the "Tool" columns of the tables. Consider switching to "working document" here.

When -> Once